addons/sys_core/fnc_processDirectSpeaker.sqf

```diff
@@ -49,16 +49,25 @@ if (_bothSpectating || {_isIntercomAttenuate}) then {
     };
 };
 
+private _zeusAdjustments = if (EGVAR(sys_zeus,zeusCommunicateViaCamera) && {call FUNC(inZeus)});
```

Comment:
```suggestion
private _zeusAdjustments = EGVAR(sys_zeus,zeusCommunicateViaCamera) && {call FUNC(inZeus)};
```
Probably a mistake?

Reply:
Not a mistake

Reply:
but why?

Reply:
I'm creating the `IfType` only once then reusing it since the if statements are all the same, doesn't matter a whole lot here but it is ever so slightly faster

Reply:
I still don't quite follow it. Isn't what jonpas suggests better and clearer to read and interpret since a basic type is stored?

Reply:
I can switch it then